FirebaseCore/Sources/FIRApp.m

```diff
@@ -573,10 +573,11 @@ - (void)checkExpectedBundleID {
 #pragma mark - private - App ID Validation
 
 /**
- * Validates the format and fingerprint of the app ID contained in GOOGLE_APP_ID in the plist file.
- * This is the main method for validating app ID.
+ * Validates the format and hashed bundle ID of the app ID contained in GOOGLE_APP_ID in the plist
```

Comment:
I wonder if we should consider omitting `and hashed bundle ID` since we can't actually check that the hash is valid client-side, just that it looks like a hash.

That said, it may be easier to differentiate the methods `validateAppIDFormat:withVersion:`
and `validateBundleIDHashWithinAppID:forVersion:` with the way you currently have it written. I'll defer to Paul for a second opinion but otherwise LGTM. Thanks!

Reply:
How about "Validates the format of app ID and its included bundle ID ...."?

Reply:
That sounds good to me. I'd just make sure to put "and its included bundle ID *hash*" since the bundle ID itself isn't included in the format.

Reply:
Thanks y'all. Done!